src/components/query-viewer/query-viewer.tsx

```diff
@@ -661,35 +696,31 @@ export class QueryViewer {
     }
   }
 
-  // ToDo: complete this implementation
   /**
    * Returns an XML on a string variable containing all the data for the attributes loaded in the Pivot Table.
    */
-  // @Method()
-  // async getFilteredData() {
-  //   if (!this.controller) {
-  //     return;
-  //   }
-  //   switch (this.RealType) {
-  //     case QueryViewerOutputType.Card:
-  //       // ToDo: implement this method to the output card
-  //       return null;
-  //     case QueryViewerOutputType.Chart:
-  //       // ToDo: implement this method to the output chart
-  //       return null;
-  //     case QueryViewerOutputType.Map:
-  //       // ToDo: implement this method to the output map
-  //       return null;
-  //     default: // PivotTable and Table
-  //       // ToDo: complete this implementation
-  //       // const serverData = this.pivotRenderRef.getPivottableDataSyncXml;
-  //       // return this.pivotRenderRef.getFilteredDataPivot(serverData);
-  //       return null;
-  //   }
-  // }
+  @Method()
+  async getFilteredData() {
+    if (!this.controller) {
+      return;
+    }
+    switch (this.type) {
+      case QueryViewerOutputType.Card:
+        // ToDo: implement this method to the output card
+        return null;
+      case QueryViewerOutputType.Chart:
+        // ToDo: implement this method to the output chart
+        return null;
+      case QueryViewerOutputType.Map:
+        // ToDo: implement this method to the output map
+        return null;
+      default: // PivotTable and Table
+        const serverData = await this.pivotRenderRef.pivotTableDataSyncXml;
```

Comment:
StencilJS' properties are readed in sync.

```suggestion
        const serverData = this.pivotRenderRef.pivotTableDataSyncXml;
```